tests/beaker_tests/snmp_notification_receiver/snmp_notification_receiver_provider_defaults.rb

```diff
@@ -53,6 +53,12 @@
 result = 'PASS'
 testheader = 'snmp_notification_receiver Resource :: All Attributes Defaults'
 
+if operating_system == 'ios_xr'
+  interface = 'gigabitethernet0/0/0/1'
```

Comment:
In my comment I was suggesting that you use the `find_interface` api to auto-discover an available interface instead of hardcoding to an interface that may or my not be available on the device.